datastream-server/src/main/java/com/linkedin/datastream/server/DatastreamTaskImpl.java

```diff
@@ -89,6 +89,7 @@
   private DatastreamEventProducer _eventProducer;
   private String _transportProviderName;
   private SerDeSet _destinationSerDes = new SerDeSet(null, null, null);
+  private int _hashCode;
```

Comment:
Can you add  a comment here or somewhere in this file that if a new field is added along with a setter, or any function that modifies the values, the setter/function must call `computeHashCode()`?